src/Integration/IODDPortReader.cs

```diff
@@ -14,7 +14,8 @@ public class IODDPortReader
     private readonly IIoddDataConverter _ioddDataConverter;
     private readonly ITypeResolverFactory _typeResolverFactory;
     private PortReaderInitilizationResult? _initilizationState;
-    private PortReaderInitilizationResult InitilizationState => _initilizationState ?? throw new InvalidOperationException("PortReader is not initialized");
+    public PortReaderInitilizationResult InitilizationState => _initilizationState ?? throw new InvalidOperationException("PortReader is not initialized");
```

Comment:
I am not so sure how much I like the idea of public exposing an internal state container. Maybe we can find a cleaner solution for this?

Reply:
Made it private again and published the IODevice property, which can be used outside and was the reason why I simply made this public ;)